packages/prisma/store/tenants.ts

```diff
@@ -0,0 +1,18 @@
+export enum Tenant {
+  US = "us",
+  EU = "eu",
+  INSIGHTS = "insights",
+}
+
+export const tenantToDatabaseUrl = {
+  [Tenant.US]: process.env.DATABASE_URL,
+  [Tenant.EU]: process.env.DATABASE_URL_EU,
+  [Tenant.INSIGHTS]: process.env.INSIGHTS_DATABASE_URL,
+};
+
+export const getTenantFromHost = (host: string) => {
+  if (host.startsWith("app.cal.eu")) {
```

Comment:
This could be a environment variable so we can test on multiple envs. Also we could opt-in to multi-tenancy.